spec/commands/inject_spec.rb

```diff
@@ -52,6 +52,29 @@
     end
   end
 
+  context "use source and group options" do
+    it "add gem with source in gemfile" do
+      bundle "inject 'bootstrap' '>0' --source=https://rubygems.org"
+      gemfile = bundled_app("Gemfile").read
+      str = "gem 'bootstrap', '> 0', :source => 'https://rubygems.org'"
+      expect(gemfile).to match(/str/)
+    end
```

Comment:
I prefer to use `include` instead if were not searching with a regex. Example:

    expect(gemfile).to include "gem 'bootstrap', '> 0', :source => 'https://rubygems.org'"

Same with the other expectations below as well.